_drafts/2021-01-08-ALTF4-20.md

```diff
@@ -0,0 +1,78 @@
+
+> If you have something interesting in mind that you want to share with the community in a polished way, this is the place to do it. If you’re not too sure about it we’ll gladly help by discussing content ideas and structure questions.
+- Therenas
+
+I think this goes to show that contributing to Alt-F4 shouldn't feel like a chore or an obligation, we want it to be a collaborative process of us helping you to tell the story you want to tell, whatever it might be about. We regularly have long conversions in our [Discord](https://discord.gg/nxnCFkb) (which you should totally join if you haven't!) with contributors about where we think they should take their pieces. For me it's a case of "I want more information about this really cool topic" and I always try to convey that enthusiasm in the hopes that we can inspire more creativity.
+
+We also do our darndest to make the barrier of entry as low as possible, making sure that anyone and everyone who wants to contribute can. We'll take submissions in whatever form they come, good or great grammar, long or short, in fancypants markdown, a text file or even handwritten! Everyone is welcome to contribute and I encourage all of you to think about if there's anything at all you want to write about, and if there is, come and chat with us about, we'll be happy to have you! I've personally never particularly enjoyed writing but writing for Alt-F4 in [release #15](https://alt-f4.blog/ALTF4-15/) was great fun and I fully intend to write more in the future. For me, it makes it so much easier writing about something you are passionate about and I think we're all passionate about this silly little factory game.
+
+## Translating Alt-F4 <author>nanogamer7</auhtor>
+
+The beginnings were small. One man, one issue, one language. But with the next issue there were already a dozen people eager to help. Looking desperately for something to do, that wasn't already someone else' job, the suggestion of translations came. With a multinational community there were quickly a few interested translators for some major and minor languages, like german, which I will focus on today. A single person has quite obvious limitations, for example the workload can't be split up, and while reviewing your translation definitely helps with quality, having more eyes look at it provides a far greater value. But multiple people pose interesting challenges of their own. First though I'd like to give you an idea of how much translation we actually do.
+
+[graph, x=*week*, y=*translated text (total characters)*; I'd like to get time of translation publish from GitHub PRs, not week of associated issue; Graph as lines, with either one line for english and one for total translated content, or one for english and one for every other language, or both]
```

Comment:
general tag about all graphics that have to be done there. will need to adjust description/caption and eventually text around